packages/medusa/src/models/sales-channel.ts

```diff
@@ -0,0 +1,23 @@
+import { BeforeInsert, Column } from "typeorm"
+
+import { SoftDeletableEntity } from "../interfaces"
+import { FeatureFlagEntity } from "../utils/feature-flag-decorators"
+import { resolveDbType } from "../utils/db-aware-column"
+import { generateEntityId } from "../utils"
+
+@FeatureFlagEntity("sales-channels")
+export class SalesChannel extends SoftDeletableEntity {
+  @Column()
+  name: string
+
+  @Column({ nullable: true })
+  description: string
+
+  @Column({ type: resolveDbType("timestamptz"), nullable: true })
+  disabled_at: Date | null
```

Comment:
If we don't have an explicit use for the date at the current time I think the boolean is sufficient. As long as we keep the api stable we can always add the date at a later time and a status in the same way we have added status to the batch jobs.

Reply:
It can be useful to display the date of when it has been disabled by the user, that gives an extra information to the user. But for now if we don’t want that info lets stick on the boolean